src/lib/containers/EntityBadgeIcons.tsx

```diff
@@ -109,8 +110,8 @@ export const EntityBadgeIcons = (props: EntityBadgeIconsProps) => {
 
   const { data: bundle } = useGetEntityBundle(
     entityId,
-    undefined,
     versionNumber,
+    ALL_ENTITY_BUNDLE_FIELDS,
```

Comment:
Reordered the arguments for `useGetEntityBundle`.

It was `(entityId, fields, versionNumber, queryOptions)`, now it's `(entityId, versionNumber, fields, queryOptions)`